### ISSUE TITLE
The solver input Remesher command does not parse correctly

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/svFSI#90]
The solver input file `Remesher` command does not parse correctly. For  example
```
   Remesher: Tetgen { 
   Max edge size:                fluid_volume-mesh-complete { val: 0.7 }
   Min dihedral angle:           10
   Max radius ratio:             1.1
   Remesh frequency:             1000
   Frequency for copying data:   5
    }
```
fails with `ERROR: Near LINE 130 at <Remesher: Tetgen>, searched COMMAND <Max edge size> :: Command not found
`.

The problem is in the `READRMSH` subroutine where the code is trying parse the `Remesher` command mesh name using
```
lPtr => lPR%get(ctmp,"Max edge size",iM)
```
this should be
```
lPtr => lPR%get(ctmp,"Max edge size")
```

@vvedula22 This is such a basic bug so I am wondering if the remeshing code has ever been tested? There are no tests in svFSI-Tests.

**Comment by vvedula22** [comment]
@ktbolt Are you using an older version of the code? The latest code uses the above format where users are allowed to remesh multiple meshes if needed. It seems to be working for others too. See this [post](https://simtk.org/plugins/phpBB/viewtopicPhpbb.php?f=188&t=15063&p=0&start=0&view=&sid=749693d9385aa2bb431079a39d6f7b37)

I have used remesher for ventricular blood flow simulations, however, the code for projecting data from old mesh to the new ones needs to be optimized. This was added during the early part of my postdoc when I was still learning the code.
